ocfweb/docs/docs/services/web.md

```diff
@@ -11,6 +11,49 @@ where `user` is the account name.
 Group accounts may also request a [[virtual host|doc services/vhost]] for
 another domain (e.g., group.berkeley.edu).
 
+#### Off-site hosting
+
+One question that often comes up is if the OCF is able to host websites
+from services such as Squarespace, Wix, Weebly, Tumblr, etc. Unfortunately
+due to logistical considerations and University policy, the OCF is unable
+to do so.
+
+First, many website building services like Wix and Weebly make their money
+by using a proprietary service to allow users to build their sites, and
+subsequently charging users to host those sites. This means providing the
+servers, storage space, internet connection, and configurations to allow
+these sites to show up on the internet. Obviously, the OCF is unable to
+access these proprietary backends and is therefore unable to host sites
+developed using them. However, the OCF is capable of hosting anything made
+using the free and open-source backends we support, including Wordpress,
+Django, Ruby on Rails, Node.js, PHP, and others. You may wish to consider
+building your website with those if you desire to host with the OCF.
+
+Second, University policy prohibits campus DNS from redirection to off-site
+hosts without explicit permission from the University. This means that, if
+you want to host your website with Wix or some other provider, we cannot
+give you a URL like group.berkeley.edu that points to that website. At the
+very least, the OCF is not the appropriate point of contact in such cases -
+one would have to
+[contact the university](https://offsitehosting.berkeley.edu/) directly
+to request permission. This is primarily relevant to student groups.
+Similarly, the OCF does not generally allow hosting under custom domain
+names. This means we do not support things like "you_or_your_group.com"
```

Comment:
This is not entirely true, at least we host a few of these non-berkeley.edu domains and we have [a paragraph](https://www.ocf.berkeley.edu/docs/services/vhost/#h4_limitations-on-non-berkeleyedu) about hosting without a berkeley.edu subdomain, although given that regular subdomains require an application anyway, I think that could be revised to essentially say only the second point that says we don't pay for the domain, the group wanting to host with us does.

I think we need a clearer policy on what to do about these, and I don't see a problem with the OCF hosting them as long as they follow our requirements.